playbooks/openstack-osh_hostconfig.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: soc-deployer
+  become: yes
```

Comment:
?

Reply:
Those tasks need to run as root. But the terraform deployed nodes don't allow ssh logins for root. That why we set  `ansible_user' to `sles` in the inventory `become: yes` here to run the task as root.
I was hoping the commit message would explain it good enough. Seems it didn't :/

Reply:
ah, I missed the commit message completely so thats my fault!